docs/sources/tempo/getting-started/metrics-from-traces.md

```diff
@@ -28,15 +28,15 @@ Span metrics are of particular interest if your system is not monitored with met
 Metrics generation is disabled by default. Contact Grafana Support to enable metrics generation in your organization.
 {{% /admonition %}}
 
-After the metrics generator is enabled in your organization, refer to [Metrics-generator configuration]({{< relref "../configuration" >}}) for information about metrics-generator options.
+After the metrics-generator is enabled in your organization, refer to [Metrics-generator configuration]({{< relref "../configuration" >}}) for information about metrics-generator options.
 
-<p align="center"><img src="../assets/trace_service_graph.png" alt="Trace service graph"></p>
+![Trace service graph](/media/docs/grafana/data-sources/tempo/query-editor/tempo-ds-query-service-graph.png)
 
 These metrics exist in your Hosted Metrics instance and can also be easily used to generate powerful custom dashboards.
 
 <p align="center"><img src="../assets/trace_custom_metrics_dash.png" alt="Trace custom metrics dashboard"></p>
 
-The metrics generator automatically generates exemplars as well which allows easy metrics to trace linking. [Exemplars](/docs/grafana-cloud/monitor-infrastructure/traces/exemplars/) are available in Grafana Cloud so you can also push your own.
+The metrics-generator automatically generates exemplars as well which allows easy metrics to trace linking. [Exemplars](https://grafana.com/docs/grafana/latest/fundamentals/exemplars/) are available in Grafana Cloud so you can also push your own.
```

Comment:
https://github.com/knylander-grafana/tempo-doc-work/blob/updates-for-query-editor-11-2/docs/sources/tempo/_index.md?plain=1#L7

```suggestion
The metrics-generator automatically generates exemplars as well which allows easy metrics to trace linking. [Exemplars](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/fundamentals/exemplars/) are available in Grafana Cloud so you can also push your own.
```

Reply:
Are you asking if we should have the grafana version defined as next or as latest? What do the other repos do for databases?

Reply:
I think that primarily links should use the version substitution if there's cross product version compatibility so that you can update all the links in one place and allow backports that work across versions.

In terms of what the value should be, I think that's probably a decision on a per project basis.
Like should the "latest" version of Tempo match the "latest" version of Grafana, should older versions also link to "latest" or perhaps there's a specific version of Grafana that Tempo docs make sense to link to.

Reply:
In this case, the page we're linking to is conceptual information, so not Grafana version specific. There might be specific versions in other docs that would matter. Wow, this could get very complicated quickly.

Reply:
For now, I don't think we have to worry about the versions. However, I'll create a doc issue to evaluate the Grafana version, Tempo version, and the intersection of features and versions for both products.

Reply:
I created a doc issue to update the URLs - https://github.com/grafana/tempo/issues/4013 and an issue to evaluate the version links.